packages/form-js-editor/src/features/properties-panel/entries/InputKeyValuesSourceEntry.js

```diff
@@ -11,20 +11,10 @@ export default function InputKeyValuesSourceEntry(props) {
     id
   } = props;
 
-  const schema = '[\n  {\n    "label": "dollar",\n    "value": "$"\n  }\n]';
-
-  const description = <div>
-    Define which input property to populate the values from.
-    <br /><br />The input property may be an array of simple values or alternatively follow this schema:
-    <pre><code>{schema}</code></pre>
-  </div>;
-
   return [
     {
       id: id + '-key',
       component: InputValuesKey,
-      label: 'Input values key',
```

Comment:
Removing the label from the `entries` configuration would make it impossible to override via extension (custom properties provider) I think?

Not sure whether this is a widely used pattern in our entries, though.

Reply:
I looked around and it's one or the other. I mostly changed this one to be in line with the Expressions equivalent which did define the label on the component itself.

Reply:
I think when we know that this matters, we can fix it. Because there's a lot of inconsistency at the moment.

Reply:
If it's not widely used, we can consider this as not-supported at the moment.

Happy to clean this up at all places in one of the next spring cleanings 🧹